content/en/security/cloud_siem/entities_and_risk_scoring.md

```diff
@@ -1,62 +1,58 @@
 ---
-title: Risk Based Entity Insights for AWS
+title: Risk Insights for AWS Entities
 further_reading:
     - link: "https://www.datadoghq.com/blog/risk-prioritization-entity-analytics/"
       tag: Blog
       text: "Accelerate investigations with Datadog Cloud SIEM Risk-based Insights and AWS Entity Analytics"
 ---
 
-{{< callout url="https://datadoghq.com/private-beta/risk-based-entity-insights" header="Join the Beta!">}}
-Risk Based Entity Insights is in private beta.
-{{< /callout >}}
-
 ## Overview
 
-[Cloud SIEM's Risk Based Entity Insights for AWS][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
+[Cloud SIEM's Risk Insights for AWS Entities][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
 
-With Risk Based Entity Insights, you can:
+With Risk Insights, you can:
 
-- Explore entities, filtering them by attributes such as [risk score](#risk-scoring) or entity type.
+- Explore entities, filtering them by attributes such as [risk score severity](#risk-scoring) and configuration risks.
 - View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
 - Triage relevant items in bulk.
 - Take mitigation steps such as creating a global suppression or creating a case for an entity.
 
 ## Prerequisites
 
-- For AWS entities coverage, [AWS must be configured for Cloud SIEM][1].
-- To view associated Cloud Security Management (CSM) data in the entity panel, [CSM must be configured][2].
+- For AWS Risk Insights coverage, [AWS must be configured for Cloud SIEM][1].
+- To view associated Cloud Security Management (CSM) insights in the entity panel, [CSM must be configured][2].
```

Comment:
```suggestion
- (Optional) To view associated Cloud Security Management (CSM) insights in the entity panel, [CSM must be configured][2].
```

Reply:
I recommend labeling CSM as being optional. That'll allow us to delete the part that comes after this, as it's just repeating the same information (that CSM is not required).